docs/src/pages/components/snackbars/snackbars.md

```diff
@@ -32,6 +32,14 @@ Here are some examples of customizing the component. You can learn more about th
 ## Positioned snackbars
 
 There may be circumstances when the placement of the snackbar needs to be more flexible.
```

Comment:
It might be worth reiterating the Material guidelines before suggesting an alternative"
```suggestion
In wide layouts, snackbars can be left-aligned or center-aligned if they are consistently placed on the same spot at the bottom of the screen, however there may be circumstances where the placement of the snackbar needs to be more flexible.
```